spec/augeas_tree_spec.rb

```diff
@@ -87,6 +87,21 @@
     end
   end
 
+  describe "#select" do
+    it "selects entries according to passed block" do
+      matcher = CFA::Matcher.new(collection: "spec")
+      expect(tree.select(matcher).size).to eq 2
```

Comment:
Well, yes, this part I understand. I meant to say that it is very hard to verify that the test does the right thing, because you have to look up the fixture config file, *and* the lens file (which is located completely elsewhere), then understand the rather complex lens.